samples/boards/nrf52/mesh/onoff_level_app/README.rst

```diff
@@ -0,0 +1,132 @@
+.. _bluetooth-mesh-onoff-level-sample:
+
+Bluetooth: Mesh OnOff & Level Model
+###################################
+
+Overview
+********
+
+This is a simple application demonstrating a Bluetooth mesh node.
```

Comment:
It would be better not to have so many blank lines (for readability):

    This is a simple application demonstrating a Bluetooth mesh node.
    The node's root element has the following models:

    - Generic OnOff Server
    - Generic OnOff Client
    - Generic Level Server
    - Generic Level Client 

    The node's secondary element has the following models:

    - Generic OnOff Server
    - Generic OnOff Client